examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
There are wider problems is BIND_OVERALL isn't functioning correctly, these need to be looked into and resolved.  Just checked GLES3 specs and the required glVertexAttrib4fv function is there:

https://www.khronos.org/registry/OpenGL-Refpages/es3.0/html/glVertexAttrib.xhtml

It is worth noting that I am current working on changes to osgText that will provide better out of the box support for osgText on GL3 and GLES2/3.  It may be that all your custom code for osgText will be not required.  Keep an eye on the textrefactor_3 branch.

For this PR I would suggest removing the shader/color/text specific changes from the PR so there aren't any items that are controversial so it can be merged as is.  We then can work towards resolving the final issues, such as making sure the new osgText branch works fine.